app/controllers/api/v1/reviews_controller.rb

```diff
@@ -21,7 +21,7 @@ def create
 
```

Comment:
what happens when I'm not signed in?

We should check in our controller to ensure that a user is signed in prior to allowing allowing them to invoke the create action or the destroy action

Check out the article on authorization: https://learn.launchacademy.com/lessons/devise-authorization